src/native/hash/bitfn.h

```diff
@@ -22,8 +22,7 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef BITFN_H
-#define BITFN_H
+#pragma once
```

Comment:
C99 [defines](http://port70.net/~nsz/c/c99/n1256.html#6.10.6) `#pragma` directives. But not  `#pragma once`.